build/uno.winui.common.targets

```diff
@@ -8,6 +8,18 @@
 		<_IsUnoPlatform>true</_IsUnoPlatform>
 	</PropertyGroup>
 
+	<!-- Feature configuration -->
+	<PropertyGroup>
+		<UnoDragDropExternalSupport Condition="'$(UnoDragDropExternalSupport)'==''">true</UnoDragDropExternalSupport>
```

Comment:
Should we use != 'false' here ?

Reply:
The point here is to provide a default value. If it's not defined, it's enabled, it otherwise uses the existing value.